R/get_recall_data.R

```diff
@@ -0,0 +1,158 @@
+#' get_recall_make
+#'
+#' @param make A string
+#' @param limit An integer
+#' @param partial A bool
+#'
+#' @return dataframe
+#'
+#' @examples
+#' \dontrun{
+#' get_recall_make('Nissan')
+#' }
+get_recall_make <- function(make, manufacturer = FALSE, limit = 25, partial = FALSE) {
```

Comment:
this seems a little strange
i.e make covering make and manufacturer

perhaps `get_recall_by_name` ?

Reply:
One can make an api call on manufacturer, make, or model. However, manufacturer and make appear to be the same across all the tests I looked at (although this will be easier to see when we have a properly formatted dataframe, if they are regularly different, then should probably have another function call). I did the function this way so that a user had the option of using manufacturer instead of make if they needed to (if the values were indeed different in some instances) w/out having another complete function for it. `get_recall_by_name()` wouldn't work as we have `get_recall_by_model()` which is definitely usefull/distinct.

Alternatively, we could wrap the `get_recall_by_model()` into this one and call it `get_recall_by_name()` and have the user select on of `c('manufacturer', 'make', 'model')`, that might be better. I'm maybe learning towards this case?

Reply:
I misread this the first time around, but I'm still confused.
I see why you did it this way now.

But,
Why do they have manufacturer and make apis, they look to be exactly the same after playing with the website for 2 mins.
Am I missing a difference?

Reply:
exactly, couldn't tell the difference between manufacturer and make after playing around BUT that doesn't mean that aren't cases where it's different so wanted to leave the functionality in to allow a user to select one or the other if needed but didn't want to create a new function so, options are:
1. leave as is in code currently
2. create a new function `*_manufacturer_*()`
3. just use `get_recall_by_name()` and allow the user to select one of `c('manufacturer', 'make', 'model')` to pass into the function as an arg

Reply:
Discussed and decided to leave as option #1 for now.